inlanecruising_plugin/src/inlanecruising_plugin.cpp

```diff
@@ -701,7 +704,7 @@ bool InLaneCruisingPlugin::validate_yield_plan(const cav_msgs::TrajectoryPlan& y
   {
     ROS_DEBUG_STREAM("Invalid Yield Trajectory"); 
   }
-  return false;
+  return true;
```

Comment:
The line right above this states this is an invalid plan and yet we are returning true???

Reply:
Agreed, not sure I understand the intent of this change, it appears that in the original code this was a fallthrough case to return false, which makes sense.